### ISSUE TITLE
Ignore duplicated attributions + fire change event once the UI element has been updated

### ISSUE THREAD

**Issue by aberenyi** [openlayers/openlayers#5297]
Hi,

Currently duplicated attributions cannot be filtered out easily.

Additionally the attribution control's `change` event is not fired once the control's UI element has been update.

Cheers

**Comment by tschaub** [comment]
Your change in eb4f4327055b7fec6bd2ce94d7f517cdae93bf97 looks sensible to me @aberenyi (though I didn't review in detail).

**Comment by tschaub** [comment]
(If the linter rules don't catch the inconsistent brace style in eb4f4327055b7fec6bd2ce94d7f517cdae93bf97, then we need to update our config.)

**Comment by fredj** [comment]
> (If the linter rules don't catch the inconsistent brace style in eb4f432, then we need to update our config.)

The brace style is correctly catch by eslint:
```
/home/fredj/code/ol3/src/ol/control/attribution.js
  190:9   error  Opening curly brace does not appear on the same line as controlling statement  brace-style
  192:11  error  Opening curly brace does not appear on the same line as controlling statement  brace-style
  194:13  error  Opening curly brace does not appear on the same line as controlling statement  brace-style
  200:9   error  Opening curly brace does not appear on the same line as controlling statement  brace-style
```

**Comment by ahocevar** [comment]
@aberenyi Would you be able to fix the brace style and create a pull request? Thanks.

**Comment by aberenyi** [comment]
@tschaub @fredj @ahocevar thanks guys for the quick response, I'll submit a PR later today

**Comment by aberenyi** [comment]
Gents, I've got 2 failing tests, however, I'm quite sure that they are not related to this.

```
  1) ol.format.GeoJSON #writeGeometry accepts featureProjection:

      expected [ 10, 19.999999999999986 ] to sort of equal [ 10, 20 ]
      + expected - actual

       [
         10
      -  19.999999999999986
      +  20
       ]
```

```
  2) ol.format.GeoJSON #writeGeometry respects featureProjection passed to constructor:

      expected [ 10, 19.999999999999986 ] to sort of equal [ 10, 20 ]
      + expected - actual

       [
         10
      -  19.999999999999986
      +  20
       ]
```

TL;DR; Full build log [here](https://gist.github.com/d44926018ff438563ef3b23782dce7bf)

**Comment by bartvde** [comment]
Same for me see: https://github.com/openlayers/ol3/pull/5917#issuecomment-253225880

**Comment by tschaub** [comment]
I think this was fixed by #6230.  Please open a new issue if there are still problems.
